### DISCUSSION TITLE
[Feature] stagetransitionscene

### DISCUSSION THREAD

**Discussion by jenngra505** [ikemen-engine/Ikemen-GO#911] [Feature Request] [answered]
This feature would allow a .def file to be loaded after all characters are selected to bring up a custom stage select screen or something more advanced. This feature exists in TMNTxJL but not in baseline IKEMEN.
Example: stagetransitionscene=data/stageselect.def

**Comment by Sluigi123** [comment]
Is that similar to how it worked on the Kenya Stage from SF3: [New Generation/Double Impact]? If so (or in the ballpark), I'm down for it!

**Comment by Paul45mec** [comment]
I would like to know how to implement that, 

i tried to see the files Start.lua and storyboard.lua 

i add the command line

[Select Info]
stagetransitionscene=data/stageselect.def

if (esc() or (attract and main.credits > 0) or (not attract and main.f_input(main.t_players, {'pal','s', 'm'})) and t.scenedef.skipbutton > 0) and fp ~=motif.select_info.stagetransitionscene then

but at least for me it doesn't work yet, frustrating because it's really exciting and needs a tutorial how to use correctly.

**Reply by Paul45mec** [reply]
I got to add this feature using the Wreq/Kamekaze code, but only i don't how to put a delay to avoid when the 2 player is selected to have his chosen animation interrupted by the transition scene, with the delay it could be solved, but i don't know how to implement it yet.

**Comment by jenngra505** [comment, marked as answer]
This pull request added features that make this possible using Lua modules; this is no longer needed on the engine-side. https://github.com/ikemen-engine/Ikemen-GO/pull/3546
